projects/ion/src/lib/breadcrumb/breadcrumb.component.scss

```
@@ -0,0 +1,68 @@
+@import '../../styles/index.scss';
+
+.ion-breadcrumbs {
+  text-decoration: none;
+  font-family: sans-serif;
+  font-size: 15px;
+}
+
+.ion-breadcrumbs__list {
+  display: inline-flex;
+  gap: 8px;
+  margin: 0;
+  padding: 0;
+}
+
+.ion-breadcrumbs__list li {
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  text-align: center;
+
+  ion-icon {
+    ::ng-deep svg {
+      fill: $neutral-5;
+    }
+  }
+}
+
+.ion-breadcrumbs__list li button {
```

You can improve this whole stylesheet with nesting now that BEM is applied. 
```suggestion
.ion-breadcrumbs {
  text-decoration: none;
  font-family: sans-serif;
  font-size: 15px;

&__list {
  display: inline-flex;
  gap: 8px;
  margin: 0;
  padding: 0;
}

&__list li {
  display: flex;
  gap: 8px;
  align-items: center;
  text-align: center;

  ion-icon {
    ::ng-deep svg {
      fill: $neutral-5;
    }
  }
}

&__list li button {
```
Obs: the nesting isn't perfect set because of code limitations here, but i believe it works for you to get the idea.